### ISSUE TITLE
I have a question.

### ISSUE THREAD

**Issue by PereverzevIvan** [LordOfPolls/Rule34-API-Wrapper#26]
Hello, I have a question: can I change the image save path? By the way, the API you developed is very cool, thank you for it!

**Comment by TimErmolt** [comment]
Yes, that is possible. You can specify a custom path as a second argument for `download()` after the image URL:

`rule34.download(url, "C:\Your\Path\Here")`

It actually doesn't work right now, always creating an error due to a bug, but after #28 (or an equivalent fix) is merged it should function properly.
